tox.ini

```diff
@@ -1,180 +1,47 @@
 [tox]
-envlist = flake8,
-          py26twisted,
-          py26asyncio,
```

Comment:
gah yes
I wanted to remove py26-twtrunk, as Twisted trunk no longer supports 2.6. Will fix.